controllers/articles-controller.js

```diff
@@ -23,3 +23,14 @@ exports.getArticles = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getComments = (req, res, next) => {
+  const { article_id } = req.params;
+  selectComments(article_id)
+    .then((comments) => {
+    res.status(200).send(comments)
```

Comment:
Make sure you are returning your data on an appropriate key e.g. {comments: comments}. You may need to tweak your tests to reflect this too.